src/LibChorus/Utilities/ExecutionEnvironment.cs

```diff
@@ -46,6 +46,8 @@ internal static string ChorusMergeFilePath()
 		{
 			if (Platform.IsMono)
 			{
+				//todo can we use ChorusMerge.dll directly? instead of chorusmerge
```

Comment:
I think when running with net6.0 we won't come into this branch, so it's only when running with net461. And in that case we have `ChorusMerge.exe`, not `ChorusMerge.dll`. And on Linux when running on Mono you need the prepended `mono `.

Reply:
We shouldn't be running with Mono on Linux anymore. Removing Mono was a major part of the reason to upgrade to .NET6. With .NET6 on Linux, this was pointing directly to the .exe instead of chorusmerge. This change points it back to chorusmerge, and I'm trying to test it

Reply:
Sure. But remember that we're still building for net461, and there very likely will be applications that will still run it with Mono, so that option still needs to work.

Reply:
If Platform.IsLinux isn't equivalent, I could also do (Platform.IsLinux || Platform.IsMono)